setup.py

```diff
@@ -23,7 +23,7 @@
     long_description += "\n\n" + f.read()
 
 
-VERSION = "0.12.0"
+VERSION = "0.12.1"
```

Comment:
```suggestion
VERSION = "0.13"
```

PATCH version bumps is only for (minor) bugfixes, this is closer to a new feature, so a MINOR version bump is more appropriate: https://semver.org/